src/client/src/rt-platforms/openFin/adapter/window.ts

```diff
@@ -99,12 +99,15 @@ export const openDesktopWindow = async (
     }
   }
 
+  const windowName = config.name || generateRandomName();
+  console.info(`Creating Openfin window: ${windowName}`);
+
   //TODO: move to openfin V2 version (based on promises) once they fix their bug related to getting current window
   // (in V2 call to ofWindow.getWebWindow() returns undefined - thus we are forced to use old callback APIs)
```

Comment:
do we need to follow this up with OpenFin @sizov ? Have you spoken to them about it at all?

Reply:
I haven't spoken about it. How would it be better to reach them out on this?